### PR TITLE
Fixes formatting issues with lists in event type description and bio

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Lists were disabled in the editor for the bio as we don't have a good way to display it, because we have the text centered like that: 
Screenshot 2023-03-03 at 17 33 51

Ordered and underordered list now for event type description: 
Screenshot 2023-03-03 at 17 35 19

Screenshot 2023-03-03 at 17 36 43

Screenshot 2023-03-03 at 17 36 59

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/lib/markdownIt.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export const md = new MarkdownIt("default", { html: true, breaks: true, linkify: true });
+
+export function addListFormatting(html: string) {
+  return html
+    .replaceAll("<ul>", "<ul style='list-style-type: disc; list-style-position: inside; margin-left: 12px'>")
+    .replaceAll(
+      "<ol>",
+      "<ol style='list-style-type: decimal; list-style-position: inside; margin-left: 12px'>"
+    );
```
Comment (PeerRich) [inline packages/lib/markdownIt.ts:11]: smart

**Review by PeerRich — APPROVED** [review]
LGTM

**Review by zomars — APPROVED** [review]

File: packages/lib/markdownIt.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import MarkdownIt from "markdown-it";
+
+export const md = new MarkdownIt("default", { html: true, breaks: true, linkify: true });
+
+export function addListFormatting(html: string) {
+  return html
+    .replaceAll("<ul>", "<ul style='list-style-type: disc; list-style-position: inside; margin-left: 12px'>")
+    .replaceAll(
+      "<ol>",
+      "<ol style='list-style-type: decimal; list-style-position: inside; margin-left: 12px'>"
```
Comment (roae) [inline packages/lib/markdownIt.ts:10]: why didn't you use tailwind here?

Comment (zomars) [reply packages/lib/markdownIt.ts:10]: Tailwind won't detect usage in this context as far as I know.

**Review by roae — APPROVED** [review]
looks great

File: packages/lib/markdownIt.ts (hunk 2 above)
Comment (roae) [reply packages/lib/markdownIt.ts:10]: isn't that path in the "content" config of tailwind?

**Pull request merged**